styles/card/card.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+.card-container{
+    padding: 1rem;
+}
+.card {
+  color: white;
+  width: 16rem;
+  padding: 1rem;
+  background-color: var(--secondary-color);
+}
+.card-title {
+  margin: .4rem 0;
+  font-size: 1.4rem;
+  font-weight: bold;
+}
+.card-btn {
+  text-transform: uppercase;
+  font-weight: bold;
+  padding: 0.4rem;
+  margin: .1rem;
```

Comment:
1.6px or 2px would be more suited here that 0.1rem

Reply:
done